artifacttemplates/http%3A%2F%2Fopentosca.org%2Fartifacttemplates/DockerEngine_DockerEngine-Interface-w1/source/src/main/java/org/opentosca/artifacttemplates/Constants.java

```diff
@@ -0,0 +1,19 @@
+package org.opentosca.artifacttemplates;
+
+public class Constants {
+
+    // namespace under which the SOAP service operates
+    protected static final String NAMESPACE_URI = "http://NodeTypes.opentosca.org";
```

Comment:
Shouldn't we use `http://artifacttemplates.opentsoca.org`?

Reply:
Same as below, can be changed but has to be equal to the ArtifactTemplate